src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -80,6 +83,16 @@
         RECALC
     }
 
+    /**
+     * Tagname for decoration blocks
+     */
+    private static String DECOBLOCK_TAG = "decoblocks";
+
+    /**
+     * Tag of all the blocks that need to be in the deco phase of the builder
+     */
+    protected static Tag<Block> decoItems;
```

Comment:
No. As discussed moved the actual tag instance to a dedicated class in the API which will group all of these.

Reply:
Yeah, but that's the way we do this in 1.15 everywhere (also in several PR that were approved from you in the past). In the 1.16 port of this PR we can move this there

Reply:
I think we all are now on the same page around what you're asking for Orion, with something like a ModTags module. 

I've been thinking about it overnight, and I realized we CAN NOT simply cache these returns, as they could be redefined at any time by a /reload command reloading a new datapack. 

This means that the proposed ModTags module isn't really following the rest of our Mod* pattern.

The "best" way I can currently see is a module with getters that lazy init the instances, and a dummy reload listener that clears the caches, with a complete sweep of the codebase to update to use it. Is that really a change we want to take in 1.15, or should it be done in 1.16 as a cleanup item?

Reply:
There is an event to do that. So we can add an event listener to that ModTags class that handles that.